docs/_book.yaml

```diff
@@ -25,8 +25,10 @@ upper_tabs:
     #   skip_translation: true
     #   contents:
     #   - include: /tensorboard/api_docs/python/_toc.yaml
-    - name: TF 2.0 Alpha
+    - name: TF 2.0 Beta
       contents:
+      - title: "Migrate to TF 2.0"
+        path: /tensorboard/r2/migrate
```

Comment:
Maybe let's name the file `migrate_tf2` so that when we move it up to the top level (as the TF 2.0 docs become the default) it will make more sense as a standalone name?